charts/osm/crds/meshconfig.yaml

```diff
@@ -97,13 +97,13 @@ spec:
                       type: array
                       items:
                         type: string
-                        pattern: ((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2})$
+                        pattern: ((?:\d{1,3}\.){3}\d{1,3})\/(\d{1,2})$
                     outboundPortExclusionList:
                       description: Global list of ports to exclude from outbound traffic interception by the sidecar proxy.
                       type: array
                       items:
                         type: integer
-                        pattern: ^[1-9]\d*$
+                        pattern: ^([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$ # 1 to 65535
```

Comment:
Would `minimum`/`maximum` vs. `pattern` work here?
https://kubernetes.io/docs/reference/kubernetes-api/extend-resources/custom-resource-definition-v1/#JSONSchemaProps

Reply:
It shows the types as `double`, so not certain it would produce the desired effect.

Reply:
Going to follow-up on this and address if possible in a subsequent PR.

Reply:
https://github.com/openservicemesh/osm/pull/3337